<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>suppliersDialog</class>
 <widget class="QDialog" name="suppliersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Suppliers</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/podbicon.png</normaloff>:/podbicon.png</iconset>
  </property>
  <widget class="QGroupBox" name="suppliersGroupBox">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>9</y>
     <width>901</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Suppliers</string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>881</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addRowPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add a new supplier to the table</string>
    </property>
    <property name="text">
     <string>Add Supplier</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="savePushButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>460</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Save changes and close the dialog</string>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>460</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Close the dialog, discarding any changes</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
